### DISCUSSION TITLE
broken focus in settings

### DISCUSSION THREAD

**Discussion by SergeyMy** [rustdesk/rustdesk#5217] [General]
broken focus in settings
**Deny LAN Discovery**

https://github.com/rustdesk/rustdesk/assets/131688106/71831120-a43b-4b8a-a993-14245f119897

**Comment by rustdesk** [comment]
Thanks

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/pull/5218

**Reply by SergeyMy** [reply]
confirm - problem fixed
